server/channels/store/localcachelayer/channel_layer.go

```diff
@@ -242,17 +245,29 @@ func (s LocalCacheChannelStore) GetMany(ids []string, allowFromCache bool) (mode
 	var channelsToQuery []string
 
 	if allowFromCache {
-		for _, id := range ids {
-			var ch *model.Channel
-			if err := s.rootStore.doStandardReadCache(s.rootStore.channelByIdCache, id, &ch); err == nil {
-				foundChannels = append(foundChannels, ch)
+		var toPass []any
```

Comment:
Nit: Can this (and the other `toPass` variables in this and other files) be of a specific type? In this case, `[]**model.Channel`?

Reply:
Haha, the common gotcha! No it can't :smile: 

Reason: the `doMultiReadCache` expects `[]any` because it needs to accept slice of any type. And `[]**model.Channel` can't be passed to a method that accepts `[]any` :)

The only way around is to declare `any` instead of `[]any` but then again you have to cast `any` to `[]any` just to be able to pass a typed slice. Not worth it.

Reply:
Ooooooooh, I see. That's not good :joy: